Assets/VIRTUOSO/VRTK/Source/SDK/Unity/SDK_UnityControllerTracker.cs

```diff
@@ -21,6 +25,7 @@ public class SDK_UnityControllerTracker : MonoBehaviour
         public XRNode nodeType;
         [Tooltip("The unique index to assign to the controller.")]
         public uint index;
+#if !UNITY_2019_1_OR_NEWER
         [Tooltip("The Unity Input name for the trigger axis.")]
         public string triggerAxisName = "";
```

Comment:
Should have a default value? Pick a platform, most likely?

Reply:
The axis is hand dependent, so I don't want to assign a default. However, reviewing this code myself (it was from the original VRTK stuff) I don't see it actually used anywhere. I'm going to double check with a 2017 build and if it's not used I'm removing it.

Reply:
It is used in pre-2019 versions. The value is defined in the prefab, so I am changing it to use the Left Hand versions by default and will add a comment to reflect that.